Firebase/Auth/Source/FIRUser.m

```diff
@@ -845,6 +845,9 @@ - (FIRAuthTokenResult *)parseIDToken:(NSString *)token error:(NSError **)error {
   NSData *decodedTokenPayloadData =
       [[NSData alloc] initWithBase64EncodedString:tokenPayload
                                           options:NSDataBase64DecodingIgnoreUnknownCharacters];
+  if (!decodedTokenPayloadData) {
+    return nil;
```

Comment:
Can you add a log to this since it's returning a nil result and nil error?

Reply:
Added an unexpectedResponse error.